features/attach_validtoken.feature

```diff
@@ -71,6 +71,12 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         """
         Enabling default service esm-infra
         """
+        When I verify that running `ua attach contract_token` `with sudo` exits `2`
+        Then stderr matches regexp:
+        """
+        This machine is already attached to 'UA Client Test'
```

Comment:
I think we should use a regex for the subscription name here, just in case it changes in the future